docs/interfaces/cli.md

```diff
@@ -92,6 +98,12 @@ It will exit with an error if there are any uncommitted changes in dependencies
 $ gitman update --force
 ```
 
+or to interactively overwrite changes, run:
+
+```sh
+$ gitman install --force-interactive
```

Comment:
Thanks! I think this one should say `update`?

Reply:
Sure, I think now it is correct.